coffee/src/CoffeeMachineSimulation.java

```diff
@@ -0,0 +1,212 @@
+import java.util.Scanner;
+
+public class CoffeeMachineSimulation {
+
+    private enum State {

```

Comment:
@tatevikb you can use separate enum . You can create an enum as you are creating class

Reply:
Separate enum is created. I would like to understand, what is advantage of it?